week1/leftRightScrollBtn.js

```diff
@@ -0,0 +1,13 @@
+document.addEventListener("DOMContentLoaded", () => {
+  let scrollContainer = document.querySelector(".group");
+  let scrollLeftBtn = document.getElementById("scrollLeftBtn");
+  let scrollRightBtn = document.getElementById("scrollRightBtn");
+
+  scrollLeftBtn.addEventListener("click", () => {
+    scrollContainer.scrollLeft -= 2000;
+  });
```

Comment:
요거 나같은 경우는 `< `버튼을 클릭했을 때, `scrollLeft = 0`으로 만들어서 아예 왼쪽 끝에 붙게 구현했어!

```suggestion
  scrollLeftBtn.addEventListener("click", () => {
    scrollContainer.scrollLeft = 0;
  });
```

Reply:
아 ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ그러네 0으로 해주면되네